script/flatpak/host-spawn/sources.json

```diff
@@ -0,0 +1,35 @@
+[
+    {
+        "type": "archive",
+        "url": "https://github.com/someone13574/host-spawn/archive/refs/tags/temp.zip",
```

Comment:
Temporary: DO NOT MERGE.

This should wait until [https://github.com/1player/host-spawn/pull/36](this PR) is merged upstream.